### ISSUE TITLE
Automatically assign the card to the creator

### ISSUE THREAD

**Issue by paeddi** [nextcloud/deck#3249]
**Is your feature request related to a problem? Please describe.**
In most cases when I create a new card, I want it assigned to myself.
At the moment I have to assign the cards to myself every time, which means a few extra clicks.

**Describe the solution you'd like**
Add a configuration option where each user can set who their cards are addressed to.

**Describe alternatives you've considered**
Actually up to now I don#t know any alternatives

**Comment by dodedodo** [comment]
I disagree with this feature. There's already an 'assign to me' button just 2 clicks away. On top of that, I feel like it may raise confusion in a teamwork context. *"Oh, I thought you were doing that?" "Nah, I just haven't fleshed out that card yet"*

**Comment by paeddi** [comment]
> I disagree with this feature. There's already an 'assign to me' button just 2 clicks away. On top of that, I feel like it may raise confusion in a teamwork context. _"Oh, I thought you were doing that?" "Nah, I just haven't fleshed out that card yet"_
As I described in my preferred solution, you _can set_ this option. You don't have to.
And yes, the button is two clicks away and I have to do it nearly every time and that's why I think it would be useful without harm to other users if you don't use this setting.
Furthermore, I don't think that Deck is overloaded with configuration options. ;-)

**Comment by neerro** [comment]
> 
> As I described in my preferred solution, you _can set_ this option.

I can agree If switch on/off automatically assigning will be realized as option.
